src/client.ts

```diff
@@ -3795,29 +3795,36 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
     }
 
     /**
-     * Set a user's power level.
+     * Set a power level to one or multiple users.
      * @param {string} roomId
-     * @param {string} userId
+     * @param {string|string[]} userId
      * @param {Number} powerLevel
      * @param {MatrixEvent} event
      * @return {Promise} Resolves: to an ISendEventResponse object
      * @return {module:http-api.MatrixError} Rejects: with an error response.
      */
     public setPowerLevel(
         roomId: string,
-        userId: string,
+        userId: string | string[],
         powerLevel: number,
         event: MatrixEvent,
     ): Promise<ISendEventResponse> {
+        type UsersWithPowerLevel = Record<string, number>;
         let content = {
             users: {},
```

Comment:
Wonder if you could do `users: {} as Record<string, number>,` and then not need the type assertions later

Reply:
Yes, that seems to work aswell 👍